waku.md

```diff
@@ -33,7 +34,7 @@
 
 ## Abstract
 
-This specification describes the format of Waku messages within the ÐΞVp2p Wire Protocol. This spec substitutes [EIP- 627](https://eips.ethereum.org/EIPS/eip-627). Waku is a fork of the original Whisper protocol that enables better usability for resource restricted devices, such as mostly-offline bandwidth-constrained smartphones. It does this primarily through (a) light node support and (b) historic messages (with a mailserver). Additionally, other experimental features for better scalability and DDoS resistance are under development and will be part of future versions.
+This specification describes the format of Waku messages within the ÐΞVp2p Wire Protocol. This spec substitutes [EIP-627](https://eips.ethereum.org/EIPS/eip-627). Waku is a fork of the original Whisper protocol that enables better usability for resource restricted devices, such as mostly-offline bandwidth-constrained smartphones. It does this primarily through (a) light node support, (b) historic messages (with a mailserver) and (c) `topic-interest` for better bandwidth usage. Additionally, other experimental features for better scalability and DDoS resistance are under development and will be part of future versions.
```

Comment:
`Additionally, other experimental features for better scalability and DDoS resistance are under development and will be part of future versions.`

think we can remove this.

Reply:
yes, but it doesn't make sense to remove until rate limiting is actually merged in vac, still waiting for that PR